website/docs/metrics.md

```diff
@@ -53,19 +53,39 @@ Below are the list of metrics provided by Gatekeeper:
 
 ## Webhook
 
-- Name: `request_count`
+- Name: `validation_request_count`
 
-    Description: `The number of requests that are routed to webhook`
+    Description: `The number of requests that are routed to validation webhook`
 
     Tags:
 
     - `admission_status`: [`allow`, `deny`]
 
     Aggregation: `Count`
 
-- Name: `request_duration_seconds`
+- Name: `validation_request_duration_seconds`
 
-    Description: `The response time in seconds`
+    Description: `The validation webhook response time in seconds`
+
+    Tags:
+
+    - `admission_status`: [`allow`, `deny`]
+
+    Aggregation: `Distribution`
+
+- Name: `mutation_request_count`
```

Comment:
is this introduced in v3.6? can we make a note

Reply:
These were added in Dec 2020 (#881), but I saw that they were missing from this `metrics.md`.

You can find the source at [`pkg/webhook/stats_reporter.go`](https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/webhook/stats_reporter.go)